pkgs/by-name/dr/dragonflydb/package.nix

```diff
@@ -27,13 +27,13 @@
 
 let
   pname = "dragonflydb";
-  version = "0.1.0";
+  version = "1.29.0";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = "dragonfly";
     rev = "v${version}";
-    hash = "sha256-P6WMW/n+VezWDXGagT4B+ZYyCp8oufDV6MTrpKpLZcs=";
+    hash = "sha256-70e9c82e1db30e8cddde260ccf8c0fb6f41b1ff6e1360b23879f81737948eca7";
```

Comment:
This hash is incorrect. For v1.29.0, the correct hash is `sha256-Q8GW/r4ZlAgvIgyzHeqkEPi0aiGsZxOKZhQ0VTo73P8=`; for v1.30.0 (if you would like to upgrade to that instead), the correct hash is `sha256-gsHIT98SVvmrbbFZ/MpIV+0YVV9iLrolVv3dUUGFraI=`